core/pom.xml

```diff
@@ -56,6 +77,19 @@
       <artifactId>spring-boot-devtools</artifactId>
       <optional>true</optional>
     </dependency>
+    
+        <dependency>
+            <groupId>com.google.auto.value</groupId>
```

Comment:
We don't want to add AutoValue until Lombok has been removed because they don't play well together.

Reply:
good to know.  I can use lombok in the meantime.